### PR TITLE
fix: hash link bug

### PR DESCRIPTION
**Pull request by harrysolovay**
Task labeled "#Links that just won't work!!! 😡"

### CODE REVIEW

File: client/src/utils/get-nav-height.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import {Breakpoint} from "../amplify-ui/styles/media";
+
+export const getNavHeight = (rems?: boolean) => {
+  const value = innerWidth > Breakpoint.LAPTOP * 16 ? 3 : 6.25;
```
Comment (jordanranz) [inline client/src/utils/get-nav-height.ts:4]: Comment this

File: client/src/docs-ui/page/page.tsx [hunk 2]
```diff
@@ -105,6 +105,15 @@ export class DocsPage {
     return this.getPageData();
   }
 
+  componentDidLoad() {
+    const {hash} = location;
+    if (hash) {
+      setTimeout(() => {
```
Comment (jordanranz) [inline client/src/docs-ui/page/page.tsx:111]: We need to solve this for the menu bug too. Chatting offline..

File: client/src/utils/scroll-to-hash.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import {headingIsVisible} from "./heading-is-visible";
+import {getElementTop} from "./get-element-top";
+import {getNavHeight} from "./get-nav-height";
+
+export const scrollToHash = (hash: string): void => {
+  const targets = Array.from(document.querySelectorAll(hash)).filter((e) =>
```
Comment (jordanranz) [inline client/src/utils/scroll-to-hash.ts:6]: We only care about hashes inside of page content right? Do we have to do a document query here?

File: client/src/utils/get-nav-height.ts (hunk 1 above)
Comment (jordanranz) [reply client/src/utils/get-nav-height.ts:4]: Actually let's just add an ID to the navbar then query this id. We can then use the height of the element

File: client/src/utils/scroll-to-hash.ts (hunk 3 above)
Comment (harrysolovay) [reply client/src/utils/scroll-to-hash.ts:6]: Yea––there might be multiple occurrences of the same ID

File: client/src/utils/get-nav-height.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      return cachedRemsResult;
+    } else if (cachedPxResult) {
+      return cachedPxResult;
+    }
+  }
+
+  const nav = document.getElementById("secondary-nav");
+
+  if (nav) {
+    cachedAt = innerWidth;
```
Comment (jordanranz) [inline client/src/utils/get-nav-height.ts:19]: What is this var for

**Pull request merged**